cmd/statusd/debug/debug.go

```diff
@@ -0,0 +1,107 @@
+package debug
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"net"
+	"reflect"
+
+	"github.com/status-im/status-go/geth/api"
+)
+
+const (
+	// CLIPort is the CLI port.
+	CLIPort = "51515"
+)
+
+// Debug provides a server receiving line based commands from a
+// CLI via the debugging port and executing those on the Status API
+// using reflection. The returned values will be rendered as
+// string and returned to the CLI.
+type Debug struct {
```

Comment:
Would not `DebugServer` be a better name? Or even just `Server`, as it is in its own package. Usage then - `package debug; debug.NewServer();` For clients, `debug.Server` sounds better than `debug.Debug`.

Reply:
Done.